test/async-hooks/test-graph.tcp.js

```diff
@@ -17,7 +17,7 @@ const server = net
 
 server.listen(common.PORT);
 
-net.connect({ port: server.address().port, host: server.address().address },
+net.connect({ port: server.address().port, host: '::1' },
```

Comment:
[This](https://github.com/nodejs/node/pull/14865#issuecomment-324660366) seems to be the reason. I will rebase on master.